### PR TITLE
Can now create project icons

### PR DESCRIPTION
**Pull request by TurnbulA**

### CODE REVIEW

File: app/js/addProject.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+const newProjectColours = [
+  ...document.querySelectorAll(".c-add-project__colors--button")
```
Comment (mali891) [inline app/js/addProject.js]: `--button` isn't a modifier

File: app/js/addProject.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+];
+
+newProjectColours.forEach(newProjectColour => {
+  newProjectColour.addEventListener("click", () => {
+    removeActiveColour();
+    newProjectColour.classList.toggle("isActive");
+    return newProjectColour;
```
Comment (mali891) [inline app/js/addProject.js]: Do you need to return this?

File: app/js/addProject.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  });
+});
+const formSubmitted = document.querySelector(".c-add-project__form");
+formSubmitted.addEventListener("submit", e => {
```
Comment (mali891) [inline app/js/addProject.js]: Why is this called `formSubmitted` ?

File: app/js/addProject.js [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+const projectValidation = () => {
+  const titleValue = document.querySelector(".c-add-project__input--title");
+  const projectColours = [...document.querySelectorAll(".isActive")];
+
+  projectColours.forEach(projectColour => {
+    if (projectColour.classList.contains("isActive")) {
+      return true;
+    } else {
+      alert("Select a colour");
```
Comment (mali891) [inline app/js/addProject.js]: This'll need replacing with something more graceful

File: app/js/addProject.js [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+      return false;
+    }
+  });
+  if (titleValue.value === "") {
+    alert("Fill out name");
```
Comment (mali891) [inline app/js/addProject.js]: And this

File: app/js/buildProjects.js [hunk 6]
```diff
@@ -1,6 +1,6 @@
-const projects = {
-  greenHouse: {
-    dataId: "greenHouse",
+let projects = [
```
Comment (mali891) [inline app/js/buildProjects.js]: Why did you change this?

File: app/js/buildProjects.js [hunk 7]
```diff
@@ -61,11 +60,33 @@ const renderIcon = () => {
...
+  const projectName = document.querySelector(".c-add-project__input--title")
+    .value;
+  const projectColour = document
+    .querySelector(".isActive")
+    .getAttribute("data-colour");
+  const projectDescription = document.querySelector(
+    ".c-add-project__input--description"
+  ).value;
+  const projectID = projectName.replace(/\s+/g, "");
+  const myObject = {
```
Comment (mali891) [inline app/js/buildProjects.js]: `myObject`?

File: app/js/buildProjects.js (hunk 6 above)
Comment (TurnbulA) [reply app/js/buildProjects.js]: Made the object into an array so appending it is less of a task

File: app/js/addProject.js (hunk 3 above)
Comment (TurnbulA) [reply app/js/addProject.js]: Used for when the form is submitted
